<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ipponboard</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/res/images/logo.png</normaloff>:/res/images/logo.png</iconset>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_0" stretch="1,0,0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_primary_view">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,0,0,0">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item row="0" column="3">
       <widget class="QPushButton" name="button_reset">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Blue (first last)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_name_blue">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>White (first last)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_name_white">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Mat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_mat"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Class</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_weight_class"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Round Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_time"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Weight</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_weight"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNeu"/>
    <addaction name="action_ffnen"/>
    <addaction name="actionSpeichern"/>
    <addaction name="separator"/>
    <addaction name="actionListe_importieren"/>
    <addaction name="actionListe_exportieren"/>
    <addaction name="separator"/>
    <addaction name="actionDrucken"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Ipponboard"/>
    <addaction name="separator"/>
    <addaction name="actionVisit_Project_Homepage"/>
    <addaction name="actionOnline_Feedback"/>
    <addaction name="actionContact_Author"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_SecondaryView"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionTest_Gong"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuManage">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="actionVereine"/>
    <addaction name="actionListen"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuView"/>
   <addaction name="menuManage"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>End Ipponboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionVisit_Project_Homepage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Visit Project Homepage</string>
   </property>
  </action>
  <action name="actionAbout_Ipponboard">
   <property name="text">
    <string>About Ipponboard</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionShow_SecondaryView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Secondary View</string>
   </property>
   <property name="toolTip">
    <string>Show secondary view</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionSelect_Font">
   <property name="text">
    <string>Select Font</string>
   </property>
  </action>
  <action name="actionSelect_Color">
   <property name="text">
    <string>Select Color</string>
   </property>
  </action>
  <action name="actionManage_Clubs">
   <property name="text">
    <string>Clubs</string>
   </property>
   <property name="toolTip">
    <string>Manage Clubs</string>
   </property>
  </action>
  <action name="actionReset_Scores">
   <property name="text">
    <string>Reset Scores</string>
   </property>
  </action>
  <action name="actionChange_Background">
   <property name="text">
    <string>Change Background</string>
   </property>
  </action>
  <action name="actionTest_Gong">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Test Gong</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionController">
   <property name="text">
    <string>Controller</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionLoad_Demo_Data">
   <property name="text">
    <string>Load Demo Data</string>
   </property>
  </action>
  <action name="actionNeu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Neu</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="action_ffnen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Öffnen</string>
   </property>
  </action>
  <action name="actionVereine">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Vereine</string>
   </property>
  </action>
  <action name="actionListen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Listen</string>
   </property>
  </action>
  <action name="actionListe_importieren">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Liste importieren</string>
   </property>
  </action>
  <action name="actionListe_exportieren">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Liste exportieren</string>
   </property>
  </action>
  <action name="actionDrucken">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Drucken</string>
   </property>
  </action>
  <action name="actionOnline_Feedback">
   <property name="text">
    <string>Provide Feedback (Online)</string>
   </property>
  </action>
  <action name="actionContact_Author">
   <property name="text">
    <string>Contact Author</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
